libr/core/cmd_open.c

```diff
@@ -1690,24 +1690,35 @@ static int cmd_open(void *data, const char *input) {
 			}
 			break;
 		case 'n': // "oon"
-			if ('n' == input[2]) {
-				RIODesc *desc = r_io_desc_get (core->io, core->file->fd);
-				if ('?' == input[3]) {
+			switch (input[2]) {
+			case 0: // "oon"
+				r_core_file_reopen (core, NULL, 0, 0);
+				break;
+			case ' ': //"oon file"
```

Comment:
add this to the help msg

Reply:
Actually this thing is not even working, so its probably better not to document it, and work on fixing and adding tests for this in a separate PR.